contracts/teller-network/EscrowRelay.sol

```diff
@@ -89,7 +89,7 @@ contract EscrowRelay is RelayRecipient, Ownable {
    * @param _nonce buyer's nonce
    * @param _signature buyer's signature
    */
-  function create (
+  function createEscrow(
```

Comment:
the value of `CREATE_SIGNATURE` must change from create(....)  to createEscrow(....)

Reply:
Ah, yes, good catch